src/iSparrow/utils.py

```diff
@@ -145,3 +148,47 @@ def load_module(alias: str, path: str):
     module = importlib.util.module_from_spec(spec)
     spec.loader.exec_module(module)
     return module
+
+
+def load_name_from_module(alias: str, path: str, name: str):
+    """
+    load_name_from_module Load a python module from 'path' with alias 'alias'
+
+    Args:
+        alias (str): module alias.
+        path (str): Path to load the module from
+        name (str): name to import
+    Returns:
+        module: Python module that has been loaded
+    """
+    spec = importlib.util.spec_from_file_location(alias, path)
+    module = importlib.util.module_from_spec(spec)
+    spec.loader.exec_module(module)
+    return getattr(module, name)
+
+
+def wait_for_file_completion(file_path: str, polling_interval=1) -> bool:
```

Comment:
Function that continually checks if a file still changes its size. Only way I could find with to make sure that a file is not read before it was written without making assumptions about the program that writes the file.